policyengine_us/variables/gov/states/ny/nysdps/nys_clean_heat_conE_incentive.py

```diff
@@ -0,0 +1,67 @@
+from policyengine_us.model_api import *
+
+class nys_clean_heat_conE_incentive(Variable):
+    value_type = float  # TODO: need to change?
+    entity = TaxUnit
+    label = "New York State Clean Heat incentive (con Edison)"
+    documentation = "The incentive for purchasing and installing a heat pump"
+    
+    unit = USD
+    definition_period = YEAR
+    reference = "https://cleanheat.ny.gov/assets/pdf/CECONY%20Clean%20Heat%20Program%20Manual%206%203%2024.pdf"
+    defined_for = StateCode.NY
+
+    def formula(parameters, period, tax_unit):
+        '''
+        Calculates the incentive and the cap for the NYS Clean Heat program.
+
+        Parameters:
+        - parameters (instance)
+        - period (int)
+        - tax_unit (instance)
+
+        Returns:
+        - float or tuple: For residential, a capped incentive (float). 
+          For multifamily, a tuple with the uncapped incentive (float), the cap (float), and maximum dwelling unit (int) if necessary.
+        '''
+
+        p = parameters(period).gov.states.ny.nysdps.nys_clean_heat
+        cost = tax_unit("nys_clean_heat_conE_project_cost", period)
+
+        var0 = tax_unit("nys_clean_heat_conE_family_type", period)
+        var1 = tax_unit("nys_clean_heat_conE_heat_pump", period)
+
+        # Residential
+        if var0.status == 'Residential':
```

Comment:
to access the value of the breakdown parameter will can use the following structure:
`uncapped_amount = p.amount[ny_clean_heat_family_category][x][y][z] `

Each of the values in the [] represents one of categories for the breakdown layers of the parameter 

No need for an if statement here

Reply:
Different variables are required for branching depending on whether it is 'Residential' or 'Multifamily', so I used **if** statements to handle the different cases.  (please refer to the attached decision tree). The same applies to other **if** statements.

`if family_type.status == 'Residential':`
Can I write the following statement with the name instead of the above statement with the value?
`if family_type == RESIDENTIAL:`

Reply:
I think we should have 2 files for residential vs multifamily, we can represent further breakdowns within each file